### ISSUE TITLE
[Question] Community Meeting?

### ISSUE THREAD

**Issue by fujitatomoya** [openyurtio/openyurt#1094]
**What happened**:

Nothing happened, just a question 😄 
Is there OpenYurt community meeting taking place local or global?

**What you expected to happen**:

Community meeting ? Sharing experience ? anything related to OpenYurt 👍 

**How to reproduce it (as minimally and precisely as possible)**:

Just be online 😮‍💨  ❓ 

**others**
/kind question

**Comment by rambohe-ch** [comment]
@fujitatomoya Yes, there is an OpenYurt community meeting every Wednesday at 11:00 AM(beijing time),  but the language of the meeting is still Chinese. 

The meeting info is here: https://github.com/openyurtio/openyurt#meeting

btw: in the community meeting, we are sharing the status of every SIGs in OpenYurt(https://github.com/orgs/openyurtio/projects), meanwhile, we also discuss proposals or some code details of OpenYurt.

**Comment by fujitatomoya** [comment]
> there is an OpenYurt community meeting every Wednesday at 11:00 AM(beijing time), but the language of the meeting is still Chinese.

I see, that is something I cannot do ... or I study Chinese 😄 

anyway we have some members in Shanghai, so they might be able to attend.

**Comment by rambohe-ch** [comment]
> > there is an OpenYurt community meeting every Wednesday at 11:00 AM(beijing time), but the language of the meeting is still Chinese.
> 
> I see, that is something I cannot do ... or I study Chinese 😄
> 
> anyway we have some members in Shanghai, so they might be able to attend.

@fujitatomoya welcome shanghai members to attend OpenYurt community meeting.
